Signal/src/views/OWSSystemMessageCell.m

```diff
@@ -40,21 +40,9 @@ - (instancetype)initWithFrame:(CGRect)frame
     return self;
 }
 
-- (instancetype)init
-{
```

Comment:
To be clear, I didn't have any qualms with this extra `init` method being defined as it was. In fact, it seems like it should *remain• as it was to return a useful object.

My confusion was the comment "don't init twice". 

How would that ever happen?

I guess if you wrote code like:

`[[[OWSSystemMessageCell alloc] init] init]`

But I guess the same could be said for *any* objc class. I just thought I was missing something.

No strong feelings either way, but I was just trying to understand what was being communicated.